R/calculate_silhouette_width.R

```diff
@@ -0,0 +1,76 @@
+#' Calculate silhouette width scores from an integrated SCE object
+#'
+#' @param integrated_sce The integrated SCE object
+#' @param pc_name The name that allows access to the PCs. Example: fastMNN_PCA
+#' @param frac_cells The fraction of cells to downsample to. Default: 0.8
+#' @param nreps The number of times to repeat sub-sampling procedure. Default: 20
+#' @param seed Seed for initializing random sampling
+#' @param batch_column The variable in `integrated_sce` indicating the grouping of interest.
+#'  Generally this is either batches or cell types. Default is "library_id".
```

Comment:
Matching the order of my other comment.
```suggestion
#' @param batch_column The variable in `integrated_sce` indicating the grouping of interest.
#'  Generally this is either batches or cell types. Default is "library_id".
#' @param frac_cells The fraction of cells to downsample to. Default: 0.8
#' @param nreps The number of times to repeat sub-sampling procedure. Default: 20
#' @param seed Seed for initializing random sampling
```